backend/src/api/comment/create_comment.rs

```diff
@@ -34,11 +40,51 @@ pub(super) async fn post<D: SqliteDatabase>(
     .await
 }
 
+async fn extract_alias<D: SqliteDatabase>(
+    mut text: String,
+    state: &Arc<AppState<D>>,
+) -> anyhow::Result<String> {
+    let aliases: Vec<String> = text
```

Comment:
If you have duplicate aliases in the comment we will try to replace each instance of them, even though `String::replace` will replace all matches.